apps/react/docs/docs/02-Installation/02-Option 2 Docker Compose Setup/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: installation-option-2
+title: Option 2 - Docker Compose Setup
+---
+
+# WKS Platform Installation Guide
+
+## Option 2: Running with Docker Compose Images
+
+:::info
+**🚧 This guide is currently under development 🚧**<br />
+Please note that some of the information in this guide may be incorrect or subject to change as we continue to update and refine the content. If you encounter any issues, don't hesitate to reach out for support and contribute to improving the documentation!
+:::
+
+:::warning
+**⚠️ For Development/Test Environments Only ⚠️**<br />
+The provided docker-compose files are recommended only for use in development or test environments.
+:::
+
+Welcome to the secondary installation guide for the WKS Power Platform! This guide provides an alternative installation approach using Docker Compose, tailored for those who prefer or require a different setup method. Covering everything from the initial Docker-Compose configuration to the final checks post-installation, this guide ensures a comprehensive and clear path to getting your system operational.
```

Comment:
I think the message here is: You can utilize provided docker compose files to accelerate the local development setup. Can we make that more clear?